src/test/resources/auditlog/endpoints/sink/configuration_no_default.yml

```diff
@@ -1,32 +1,33 @@
-opendistro_security:
-  audit:
-    endpoints:
-      eNDpoint1:
-        type: internal_opensearch
-        config:
-          key: value
-          wedontneed: anyconfigforinternal
-      endpoint2:
-        type: external_opensearch
-      endPOINT3:
-        type: debug
-      endpoint4:
-        type: idonotexist
-      endpoint5:
-        type: external_opensearch
-        something:
-          key: value
-      endpoint6:        
-        something:
-          key: value
-      endpoint7:        
-        config:
-          key: value
-      endpoint8:
-        type: DeBug
-        config:
-          key: value
-      endpoint9:
-        type: external_opensearch
-        config:
-          endpoints: stringhere
\ No newline at end of file
+plugins:
```

Comment:
can you do `plugins.security:` and keep everything else the same.

Reply:
Done